web/libs/editor/src/components/AnnotationTabs/AnnotationTabs.scss

```diff
@@ -0,0 +1,111 @@
+.annotation-tabs {
```

Comment:
web/libs/editor/src/components/AnnotationTabs/AnnotationTabs.styl doesnt appear to have been removed